### PR TITLE
experimental data added for ali2020

### LINKED ISSUE #17: Add Measurement Data
Add Measurement Data

### PR DESCRIPTION
**Pull request by aravinth-ravikumar**
Fixes #17

### CODE REVIEW

**Comment by aravinth-ravikumar** [comment]
Function eval_material_loss_exp() provides the experimental mass loss. @svchb you can use that for plotting.

File: data/tables/.~lock.ali2020_tables_table_4.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+,ravikuma,a0en2100015,05.09.2024 11:24,file:///home/ravikuma/.config/libreoffice/4;
```
Comment (svchb) [inline data/tables/.~lock.ali2020_tables_table_4.csv#:1]: This file should be removed

File: data/tables/ali2020_tables_table_4.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+t (h),W_(L)(mg),,,,
+%NaCl,0,2,3,4,5
+240,0.216,0.248,0.293,0.318,0.343
+480,0.32,0.426,0.446,0.449,0.569
+720,0.475,0.563,0.646,0.662,0.766
+960,0.746,0.822,0.847,0.865,0.942
+1200,0.945,1.007,1.079,1.121,1.191
+1440,1.105,1.197,1.248,1.347,1.402
+1750,1.364,1.55,1.617,1.854,2.171
```
Comment (svchb) [inline data/tables/ali2020_tables_table_4.csv:9]: Please rename all of these files this shouldn't be tables_table one table should be enough

File: web_app/model_view.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import streamlit as st
 import plotly.express as px
 import streamlit.components.v1 as components
+from matplotlib import pyplot as plt
```
Comment (svchb) [inline web_app/model_view.py:6]:
```suggestion
from matplotlib import pyplot as plt
```
pyplot is not used

File: web_app/models/IC_model_ali2020.py [hunk 4]
```diff
@@ -64,6 +76,51 @@ def eval_material_loss(self, time: float) -> float:
...
+        
+        Q11 = values[y1, x1]
+        Q21 = values[y1, x2]
+        Q12 = values[y2, x1]
+        Q22 = values[y2, x2]
+        
+        return (Q11 * (x2_val - x) * (y2_val - y) +
+                Q21 * (x - x1_val) * (y2_val - y) +
+                Q12 * (x2_val - x) * (y - y1_val) +
+                Q22 * (x - x1_val) * (y - y1_val)) / ((x2_val - x1_val) * (y2_val - y1_val))
```
Comment (svchb) [inline web_app/models/IC_model_ali2020.py:108]: why?

**Review by svchb — CHANGES_REQUESTED** [review]

File: web_app/model_view.py (hunk 3 above)
Comment (aravinth-ravikumar) [reply web_app/model_view.py:6]: no, I will remove it... it is a mistake

File: web_app/models/IC_model_ali2020.py (hunk 4 above)
Comment (aravinth-ravikumar) [reply web_app/models/IC_model_ali2020.py:108]: The experimental value is provided only for specific values, so interpolate and extrapolate to to extended time and other configurations.

File: data/tables/.~lock.ali2020_tables_table_4.csv# (hunk 1 above)
Comment (aravinth-ravikumar) [reply data/tables/.~lock.ali2020_tables_table_4.csv#:1]: done

File: data/tables/ali2020_tables_table_4.csv (hunk 2 above)
Comment (aravinth-ravikumar) [reply data/tables/ali2020_tables_table_4.csv]: done

File: web_app/models/IC_model_ali2020.py (hunk 4 above)
Comment (svchb) [reply web_app/models/IC_model_ali2020.py:103]: You can not do that. This would than bascially be a linear model in it self.

Comment (aravinth-ravikumar) [reply web_app/models/IC_model_ali2020.py:103]: True. Then the model can be compared with experimental results only till 1750 Hours. How do we proceed?

**Review by svchb — APPROVED** [review]

**Review by aravinth-ravikumar — COMMENTED** [review]
looks good

**Review by svchb — APPROVED** [review]

**Pull request merged**